grails-app/services/org/broadinstitute/orsp/NotifyService.groovy

```diff
@@ -15,6 +16,11 @@ import org.springframework.http.MediaType
 class NotifyService implements SendgridSupport, Status {
 
     public static final String ORSP_ADDRESS = "orsp-portal@broadinstitute.org"
+    public static final String YES = "Yes"
+    public static final String UNCERTAIN = "Uncertain"
+    public static final String TEXT_SHARING_BOTH = "both"
+    public static final String TEXT_SHARING_OPEN = "open"
```

Comment:
Isn't this same code in [the other PR](https://github.com/broadinstitute/orsp-pub/pull/138/files#diff-0401de3e18519430c4c05ee635382da1R23)?  Maybe this PR should wait for the other to land before requesting a review?